<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>120</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>29</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min x:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_x_line">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>120</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>148</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>r:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>317</x>
      <y>148</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>h:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="r_line">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>148</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="h_line">
    <property name="geometry">
     <rect>
      <x>333</x>
      <y>148</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Iinfo</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="fileName">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>363</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseFileButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ChooseFile</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_x_line">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>120</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>148</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>N:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="n_line">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>148</y>
      <width>118</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>178</y>
      <width>135</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="average_line">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="S_line">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Sx_line">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>178</y>
      <width>76</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Average value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>S:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sx:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>178</y>
      <width>149</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>f:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="f_line">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>149</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>X^2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xi_line">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="resultButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Check result</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>229</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Choose a:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>0,001</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0,01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0,02</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0,05</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0,10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0,20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0,80</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0,95</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0,99</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Quantile x^2  (f,1 - a)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Quantile x^2  (f,1 - a)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="quantile_line"/>
     </item>
    </layout>
   </widget>
   <zorder>fileName</zorder>
   <zorder>chooseFileButton</zorder>
   <zorder>label_8</zorder>
   <zorder>n_line</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>min_x_line</zorder>
   <zorder>max_x_line</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>r_line</zorder>
   <zorder>h_line</zorder>
   <zorder>infoButton</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>xi_line</zorder>
   <zorder>label_10</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>quantile_line</zorder>
   <zorder>resultButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
